napari/utils/colormaps/colormap_utils.py

```diff
@@ -29,20 +23,70 @@
 ]
 
 
-primary_color_names = ['red', 'green', 'blue', 'cyan', 'magenta', 'yellow']
-primary_colors = np.array(
-    [(1, 0, 0), (0, 1, 0), (0, 0, 1), (0, 1, 1), (1, 0, 1), (1, 1, 0)],
-    dtype=float,
+matplotlib_colormaps = _MATPLOTLIB_COLORMAP_NAMES = OrderedDict(
+    viridis=trans._p('colormap', 'viridis'),
+    magma=trans._p('colormap', 'magma'),
+    inferno=trans._p('colormap', 'inferno'),
+    plasma=trans._p('colormap', 'plasma'),
+    gray=trans._p('colormap', 'gray'),
+    gray_r=trans._p('colormap', 'gray r'),
```

Comment:
I was trying out this PR now and still saw `gray_r`, I was expecting to see `gray r`

![image](https://user-images.githubusercontent.com/6531703/113900030-3ffcb200-9782-11eb-8690-502c8814224d.png)

Am i missing something @goanpeca

Reply:
`viewer.layers[0].colormap.display_name` is `gray r`, so it just looks like the new name is not actually getting displayed

Reply:
Ohhh you sir have found a bug :), let me take a look!

Reply:
Fixed!